### ISSUE TITLE
help to calibrate APSIM next gen

### ISSUE THREAD

**Issue by ccnuepane** [APSIMInitiative/APSIMClassic#2263]
Hi, 

I am new to APSIM and have been working on soil organic carbon. I am currently trying to calibrate the APSIM Next Gen model against observed yield, soil organic carbon and biomass data. However, I am having trouble adjusting parameters (such as minimum leaf number, sensitivity to vernalisation, sensitivity to photoperiod, base phyllochron, and water content), as they appear as 'read-only' when I attempt to change them. Most of the examples and references I’ve found are for the classic version of APSIM, and I’m struggling to find guidance for the Next Gen model. Could anyone provide advice on how to modify these parameters or direct me to relevant resources?

Thanks,
Chandani

**Comment by peter-devoil** [comment]
Yes, most of the modules in nextgen are read only, as discussed [here](https://github.com/APSIMInitiative/ApsimX/issues/8331). It can be a nuisance.

There are ways (usually involving text editors and hand editing .apsimx files) that you can do this - you could ask how in the [issues page](https://github.com/APSIMInitiative/ApsimX/issues/)
